config/config.exs

```diff
@@ -1,8 +1,21 @@
 use Mix.Config
 
-# Import release config
-import_config "../rel/config/config.exs"
+# Configure application
+config :elixir_boilerplate, ecto_repos: [ElixirBoilerplate.Repo]
 
-# Import environment specific config. This must remain at the bottom
-# of this file so it overrides the configuration defined above.
+# Configure Phoenix
+config :phoenix, :json_library, Jason
+
+# Configure endpoint
+config :elixir_boilerplate, ElixirBoilerplateWeb.Endpoint,
+  pubsub: [name: ElixirBoilerplate.PubSub, adapter: Phoenix.PubSub.PG2],
+  render_errors: [view: ElixirBoilerplateWeb.Errors.View, accepts: ~w(html json)]
+
+# Configure Gettext
+config :elixir_boilerplate, ElixirBoilerplate.Gettext, default_locale: "en"
+
+# Import runtime configuration
+import_config "../rel/config/release.exs"
```

Comment:
I think we should be a bit more explicit and name the file `runtime.exs` since:

* The file is used to store runtime configuration
* The file is not only used to store configuration that will be used in the release — it will also be used when running `mix phx.server` in all environments